src/renderer/src/providers/ChatProvider.jsx

```
@@ -1893,19 +1910,61 @@ const useChatStore = create((set, get) => ({
+    );
+
     switch (type) {
       case "connection_established":
         break;
       case "emote_set.update":
         get().handleEmoteSetUpdate(chatroomId, body);
         break;
       case "cosmetic.create":
-        useCosmeticsStore?.getState()?.addCosmetics(body);
+      case "cosmetic.create": {
+        console.log(
```

Biome (2.1.2)

[error] Duplicate case label.

The first similar label is here:


(lint/suspicious/noDuplicateCase)